### PR TITLE
merge sdk config demo into main page

### PR DESCRIPTION
**Pull request by cb-jake**
### _Summary_

This PR merges the SDK configuration work into the main index page for more seamless testing

### _How did you test your changes?_

* yarn dev
* localhost:3001
* spin up scw locally to test the ingested values

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: examples/testapp/src/context/CBWSDKReactContextProvider.tsx [hunk 1]
```diff
@@ -133,17 +167,32 @@ export function CBWSDKReactContextProvider({ children }: CBWSDKProviderProps) {
...
-    [sdk, provider, option, setPreference, version, setSDKVersion, scwUrl, setScwUrlAndSave]
+    [
+      sdk,
+      provider,
+      option,
+      setPreference,
+      version,
+      setSDKVersion,
+      scwUrl,
+      setScwUrlAndSave,
```
Comment (nateReiners) [inline examples/testapp/src/context/CBWSDKReactContextProvider.tsx:181]: Should we add `config` to deps?

Comment (cb-jake) [reply examples/testapp/src/context/CBWSDKReactContextProvider.tsx:181]: yes. we need to linting fixed in the SDK. I have a separate task to look into it

**Review by nateReiners — APPROVED** [review]
lgtm!

**Pull request merged**